.changeset/silent-terms-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/internet-header': patch
+---
+
+Added abort controller for all existing post-search search endpoints, to avoid running parallel calls for the same endpoint and therefore get wrong answers shown in the search suggestion list.
```

Comment:
```suggestion
Added abort controller for all existing post-search search endpoints, to avoid running parallel calls for the same endpoint and therefore get outdated suggestions in the search suggestion list.
```